packages/react-table/src/components/Table/Table.tsx

```diff
@@ -93,6 +93,14 @@ export interface TableProps extends OUIAProps {
   dropdownPosition?: 'right' | 'left';
   /** The desired direction to show the dropdown when clicking on the actions Kebab. Can only be used together with `actions` property */
   dropdownDirection?: 'up' | 'down';
+  /** The container to append the dropdown menu to. Defaults to 'inline'.
+   * If your menu is being cut off you can append it to an element higher up the DOM tree.
+   * Some examples:
+   * actionsMenuAppendTo="parent"
+   * actionsMenuAppendTo={() => document.body}
+   * actionsMenuAppendTo={document.getElementById('target')}
+   */
+  actionsMenuAppendTo?: HTMLElement | (() => HTMLElement) | 'inline' | 'parent';
```

Comment:
nit. Do we need to prepend "actions" here?  The `dropdownPostiton` and `dropdownDirection` that are also related to the action do not have it prepended.  Ant there is no other `menuAppendTo` prop.

Reply:
hm... it felt clearer to me since this is a top level prop for the legacy table - so it feels so far removed from the actions menu. but I guess it's not **necessary**

Reply:
I don not feel strongly about this one. Either works.